src/fillTank.test.js

```diff
@@ -2,10 +1,152 @@
+/* eslint quotes: ["off"] */
+"use strict";
 
-describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
+describe("fillTank", () => {
+  const { fillTank } = require("./fillTank");
 
-  it('should ', () => {
+  it(`should not fill tank, because it is
+     already full (not given amount)`, () => {
+    const customer = {
+      money: 3000,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 40,
+      },
+    };
 
+    fillTank(customer, 50);
+
+    expect(customer).toEqual({
+      money: 3000,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 40,
+      },
+    });
+  });
+
+  it(`should fill tank, money is enough
+    (not given amount)`, () => {
+    const customer = {
+      money: 3000,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 30,
+      },
+    };
+
+    fillTank(customer, 50);
+
+    expect(customer).toEqual({
+      money: 2500,
+      vehicle: {
+        maxTankCapacity: 40,
+        fuelRemains: 40,
+      },
+    });
+  });
+
+  it(`should fill tank, money is not enough
+    for all capacity (not given amount)`, () => {
```

Comment:
The test description mentions 'not given amount', but the `fillTank` function is called with an amount of 50. Ensure the test description matches the test logic.